docs/community/topics/i18n.rst

```diff
@@ -1,120 +1,112 @@
+Remember to `manually escape variables <https://jinja.palletsprojects.com/en/2.11.x/templates/#working-with-manual-escaping>`__ if needed.
 
-Mark natural language text as translateable
--------------------------------------------
+For example, to mark the text "Next page" as translatable, you would write:
 
-All natural language text must be marked as translatable, so that it can be extracted by Gettext and translated by humans.
+.. code-block:: jinja
+   
+   <button type="button">
+      {{- _("Next page") -}}
+   </button>
 
-Jinja2 provides a ``trans`` block and a ``_()`` function to mark text as translatable.
-`Please refer to the Jinja2 documentation <https://jinja.palletsprojects.com/en/2.11.x/templates/#i18n>`__.
-Remember to `manually escape <https://jinja.palletsprojects.com/en/2.11.x/templates/#working-with-manual-escaping>`__ variables if needed.
 
-Any text that is marked in this way will be discoverable by ``gettext`` and used to generate ``.po`` files (see below for information).
-Once you've marked text as translateable, complete the steps for :ref:`changing-natural-language-text`.
+Any text that is marked in this way will be discoverable by ``pybabel`` and used to generate the ``PO`` files.
+Once you've marked the text as translatable, complete the steps outlined in :ref:`changing-natural-language-text`.
 
 .. _changing-natural-language-text:
 
-Add or change natural language text
------------------------------------
+Adding or changing natural language text
+-----------------------------------------
 
-These steps cover how to add or change text that has been marked as translateable.
+If you add or change natural language text in the theme, you must update the message catalogs to include the new or updated text. Follow these steps:
 
-#. Edit the natural language text as desired.
-   Ensure that it is {ref}`marked as translateable <adding-natural-language-text>`.
+#. Edit the natural language text and ensure it is :ref:`marked as translatable <adding-translatable-text>`.
 
-#. Generate/update the message catalog template (``POT`` file) with `the PyBabel extract command <https://babel.pocoo.org/en/latest/cmdline.html#extract>`__:
+#. Generate/update the message catalog template (``POT`` file):
 
    .. code-block:: bash
+      
+      tox run -e i18n-extract
 
-      pybabel extract . -F babel.cfg -o src/pydata_sphinx_theme/locale/sphinx.pot -k '_ __ l_ lazy_gettext'
-
-   **To run this in ``.nox``**: ``nox -s translate -- extract``.
-
-#. Update the message catalogs (``PO`` files) with `the PyBabel update command <https://babel.pocoo.org/en/latest/cmdline.html#update>`__:
+#. Update the message catalogs (``PO`` files) for the existing locales:
 
    .. code-block:: bash
 
-      pybabel update -i src/pydata_sphinx_theme/locale/sphinx.pot -d src/pydata_sphinx_theme/locale -D sphinx
-
-   **To run this in ``.nox``**: ``nox -s translate -- update``.
+      tox run -e i18n-update
```

Comment:
since the workflow is 

```bash
tox run -e i18n-extract
tox run -e i18n-update
```

...with apparently no steps in between, I find myself wondering why these are two separate steps (from the contributor perspective).  Why not a single tox command to do both things?

Reply:
🤔 all the sources and projects I surveyed to get an idea of the workflow treat these as a separate step... will have another thought and it might make sense to merge

Reply:
IIUC, I guess it makes sense sometimes to do `update` without doing `extract` (i.e., if only non-translatable stuff changed, and you just want to update the line numbers)?  But even so it would still be possible to have a single recipe to do both, and a sub-recipe that only did the update part.

Reply:
That makes total sense to me based on the [workflow](https://github.com/pydata/pydata-sphinx-theme/pull/1959#pullrequestreview-2266707690) I followed.